.circleci/config.yml

```diff
@@ -1,7 +1,6 @@
 orbs:
-  aws-sg-white-list-circleci-ip: inokappa/aws-sg-white-list-circleci-ip@0.0.1
-  aws-cli: circleci/aws-cli@3.1.3
-version: 2.1
```

Comment:
Did you mean to remove the version line?

Reply:
I did not, thank you.